python/setup.py

```diff
@@ -16,7 +16,7 @@
 
 setup(
     name="magma-access-gateway-installer",
-    version="1.8.0",
+    version="1.9.0",
```

Comment:
Still there :eyes:

Reply:
`main` is now meant to hold the next release. We can't be building `1.8` from both `main` and the `1.8` branch. 
For now they'll be the same, but eventually, we'll bring in 1.9 changes here.

Reply:
This version here doesn't point to any upstream release. It defines our component's version which is related to the snap version. Upstream package version that we actually install is defined in the `agw_installer.py` and it currently points to `focal-1.8.0`.